pugdebug/gui/main_window.py

```diff
@@ -10,12 +10,14 @@
 __author__ = "robertbasic"
 
 from PyQt5.QtWidgets import QMainWindow, QGridLayout, QWidget, QToolBar
-from PyQt5.QtGui import QFont
+from PyQt5.QtGui import QFont, QKeySequence
 
 from pugdebug.gui.file_browser import PugdebugFileBrowser
 from pugdebug.gui.settings import PugdebugSettingsWindow
 from pugdebug.gui.workarea import PugdebugWorkareaWindow
 
+from textwrap import fill
```

Comment:
Wraps the text to 70 chars (default value). So the tooltips are not in one line.